ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/state/beaconstate/versions/electra/BeaconStateElectra.java

```diff
@@ -35,6 +35,62 @@
 import tech.pegasys.teku.spec.datastructures.state.versions.electra.PendingDeposit;
 import tech.pegasys.teku.spec.datastructures.state.versions.electra.PendingPartialWithdrawal;
 
+/*
+ * <spec ssz_object="BeaconState" fork="electra">
+ * class BeaconState(Container):
+ *     # Versioning
+ *     genesis_time: uint64
+ *     genesis_validators_root: Root
```

Comment:
state for example is a good case of it being almost impossible to reconcile if its inline in the code imo...